src/cdk/datetime/dateadapter/date-adapter.md

```diff
@@ -0,0 +1,14 @@
+CDK `DateAdapter` component gives a foundation for more concrete material `DateAdapter` implementations.
```

Comment:
the `dateadapter` directory isn't necessary, you can just put these files directly under `datetime`.